extensions/amp-sidebar/1.0/sidebar.jss.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {createUseStyles} from 'react-jss';
+
+const sidebarClass = {
+  position: 'fixed !important',
```

Comment:
FMI: Why do we need these `!important`s?

Reply:
Mostly just so the user cannot over overwrite by accident (or on purpose).  The way the sidebar animates in and is positioned on the viewport is all critically dependent on `position: fixed`.